docs/docs/usage/scaladoc/settings.md

```diff
@@ -134,3 +142,38 @@ Example JSON file:
 }
 ```
 
+##### -snippet-compiler
+
+Snippet compiler arguments provide a way to configure snippet checking.
+
+This setting accepts a list of arguments in the format:
+args := arg{,arg}
+arg := [path=]flag
+where the path is a prefix of source paths to members to which argument should be set.
```

Comment:
I do not understand this sentence

Reply:
maybe something like `where 'path' is a prefix of the path to source files where snippets are located and 'flag' is the mode in which snippets will be type checked` would work?